Type 'unique symbol' is not assignable to type 'number'

Hei,

I was checking out Dinatra and tried using dejs. It was working for me but after upgrading to most recent deno I see the errors below, similiar to the ones from the latest [CI failure](https://travis-ci.org/syumai/dejs/builds/540882570). Do you have a fix for this?

Cool project, thanks!

```
deno run --allow-read --allow-net src/main.ts
[0/1] Compiling file:///Users/scanf/src/github.com/scanf/uuid.alemayhu.com/src/main.tshttps://raw.githubuserconte
nt.com/syumai/dejs/master/dejs.ts:99:9 - error TS2469: The '<' operator cannot be applied to type 'symbol'.

99     if (byte < 0) {
           ~~~~
https://raw.githubusercontent.com/syumai/dejs/master/dejs.ts:103:14 - error TS2345: Argument of type 'number | un
ique symbol' is not assignable to parameter of type 'number'.
  Type 'unique symbol' is not assignable to type 'number'.

103     buf.push(byte);
                 ~~~~

make: *** [hello] Error 1
Alexanders-MacBook-Pro:uuid.alemayhu.com scanf$ make
deno run --allow-read --allow-net src/main.ts
[0/1] Compiling file:///Users/scanf/src/github.com/scanf/uuid.alemayhu.com/src/main.tshttps://raw.githubuserconte
nt.com/syumai/dejs/master/dejs.ts:99:9 - error TS2469: The '<' operator cannot be applied to type 'symbol'.

99     if (byte < 0) {
           ~~~~
https://raw.githubusercontent.com/syumai/dejs/master/dejs.ts:103:14 - error TS2345: Argument of type 'number | un
ique symbol' is not assignable to parameter of type 'number'.
  Type 'unique symbol' is not assignable to type 'number'.

103     buf.push(byte);
```

--- a/dejs.ts
+++ b/dejs.ts
@@ -1,7 +1,7 @@
 const { open, Buffer } = Deno;
 import Reader = Deno.Reader;
 import { stringsReader } from 'https://deno.land/x/io/util.ts';
-import { BufReader } from 'https://deno.land/x/io/bufio.ts';
+import { BufReader, EOF } from 'https://deno.land/x/io/bufio.ts';
 import escape from 'https://deno.land/x/lodash/escape.js';
 
 const globalEval = eval;
@@ -96,7 +96,7 @@ export async function compile(reader: Reader): Promise<Template> {
 
   while (true) {
     const byte = await src.readByte();
-    if (byte < 0) {
+    if (byte === EOF) {
       break;
     }
 

